src/ontology/mondo-edit.obo

```diff
@@ -553738,10 +553740,12 @@ is_a: MONDO:0019952 {source="OMIM:620351"} ! congenital myopathy
 [Term]
 id: MONDO:0957248
 name: developmental and epileptic encephalopathy 31B
+def: "Autosomal recessive neurologic disorder characterized by early-onset epilepsy, generalized muscular hypotonia, visual impairment, and severe neurodevelopmental delay." [https://orcid.org/0000-0002-0587-4693, OMIM:620352]
```

Comment:
definition is too broad

Reply:
@yshwetar In the ontology, I do not see that this term is an "early infantile epileptic encephalopathy". In fact, "early infantile epileptic encephalopathy" was obsoleted and replaced by "developmental and epileptic encephalopathy".
I suggest that you update the definition to say: "any developmental and epileptic encephalopathy in which (...)", where "developmental and epileptic encephalopathy" is the parent term of this term.

Reply:
Comments addressed.